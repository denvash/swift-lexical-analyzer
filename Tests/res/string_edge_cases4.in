let x : String = "You should not reach this string but it has an invalid \u{0000000040} edge case that'll come up in the shuffle, since it's too many hex digits"
