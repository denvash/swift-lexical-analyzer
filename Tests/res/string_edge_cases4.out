1 LET let
1 ID x
1 COLON :
1 TYPE String
1 ASSIGN =
Error undefined escape sequence u
